Kvent/templates/kvent/profile.html

```diff
@@ -19,9 +19,14 @@
         <ul>
             <li><a href="{% url 'index' %}">Kvent</a></li>
             <li><a href="{% url 'index' %}" id="event-logo">Events</a></li>
+            {% if user.is_authenticated %}
+                <a class="btn btn-primary" href="{% url 'logout' %}">Logout</a>
+            {% endif %}
```

Comment:
You should also add this button in the index and don't forget to parse it inside <li> tag